src/api/registration/registration.service.test.js

```diff
@@ -157,14 +157,6 @@ describe("Function: saveRegistration: ", () => {
     });
   });
 
-  it("Should should call the updateStatusInCache with true", () => {
```

Comment:
Assuming Si knows these have been clipped? I assume they are superfluous now mate?

Reply:
Yep, tests removed as they are superfluous now, with the function they were testing being moved out of registration.service